### PR TITLE
De-dupe some shared UI components

### PR DESCRIPTION
**Pull request by alexchuber**
Some quick changes to reduce the number of duplication between our node editors. These were the components that were easiest / required the least amount of changing to de-dupe. The remaining duplicate components will need a closer look.

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15682/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15682/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15682/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15682/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15682/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15682/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15682/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15682/merge#BCU1XR#0

File: packages/tools/nodeEditor/src/components/propertyTab/propertyTabComponent.tsx [hunk 1]
```diff
@@ -559,7 +559,7 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
                         />
                     </LineContainerComponent>
                     <LineContainerComponent title="FILE">
-                        <FileButtonLineComponent label="Load" onClick={(file) => this.load(file)} accept=".json" />
+                        <FileButtonLine label="Load" onClick={(file) => this.load(file)} accept=".json" />
```
Comment (AmoebaChant) [inline packages/tools/nodeEditor/src/components/propertyTab/propertyTabComponent.tsx:562]: I'd like to see us standardize the naming as we clean up the duplication, but that might be best as its own "rename only" PR.  Specifically, I think we should either have Component as a suffix in all of our React components, or in none of them (unless there's a distinction I'm not seeing between the two).

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15682/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15682/merge/testResults/webgl2playwright/index.html

**Review by AmoebaChant — APPROVED** [review]

**Review by RaananW — APPROVED** [review]

File: packages/tools/nodeEditor/src/components/propertyTab/propertyTabComponent.tsx (hunk 1 above)
Comment (alexchuber) [reply packages/tools/nodeEditor/src/components/propertyTab/propertyTabComponent.tsx:562]: Agree that the naming should be standardized everywhere.

**Pull request merged**